sdk/resourcemanager/Azure.ResourceManager.Core/src/ArmClientOptions.cs

```diff
@@ -29,7 +27,7 @@ public sealed class ArmClientOptions : ClientOptions
         /// Initializes a new instance of the <see cref="ArmClientOptions"/> class.
         /// </summary>
         public ArmClientOptions()
-            : this(LocationData.Default, null)
+            : this(LocationData.Default)
```

Comment:
separate question, I think we should make Location.Default settable. Currently is fixed to WestUs.

Reply:
https://dev.azure.com/azure-mgmt-ex/DotNET%20Management%20SDK/_workitems/edit/5845